jupyter/analysis-automation/BasicExample.ipynb

```diff
@@ -0,0 +1,456 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<table>\n",
+    "    <tr>\n",
+    "        <td><img src='../SystemLink_icon.png' /></td>\n",
+    "        <td ><h1><strong>NI SystemLink Analysis Automation</strong></h1></td>\n",
+    "    </tr>\n",
+    "</table>\n",
+    "\n",
+    "his notebook is an example for how you can analyze your data with NI SystemLink Analysis Automation. It forms the core of the analysis procedure, which includes the notebook, the query, and the execution parameters (parallel or comparative). The [procedure is uploaded to Analysis Automation](https://www.ni.com/documentation/en/systemlink/latest/analysis/creating-anp-with-jupyter/). The output is a report in form of PDF documents or HTML pages. \n",
+    "<br>\n",
+    "<hr>\n",
+    "\n",
+    "## Prerequisites\n",
+    "Before you run this example, you need to [create a DataFinder search query](https://www.ni.com/documentation/en/systemlink/latest/datanavigation/finding-data-with-advanced-search/) in Data Navigation to find the example files (e.g. 'TR_M17_QT_42-1.tdms'). Save this query on the server.\n",
+    "<hr>\n",
+    "\n",
+    "## Summary\n",
```

Comment:
Consider renaming, e.g. "About this example"

Reply:
Did you mean to rename "Summary" to "About this example"?

Reply:
> Consider renaming, e.g. "About this example"

Did you mean to rename "Summary" to "About this example"?

Reply:
"Summary" is a good fit and decided to stick with it.